mappings/net/minecraft/entity/ai/brain/MemoryModuleType.mapping

```diff
@@ -1,4 +1,5 @@
 CLASS net/minecraft/class_4140 net/minecraft/entity/ai/brain/MemoryModuleType
+	FIELD field_24668 factory Ljava/util/Optional;
```

Comment:
Not sure about this, it's an optional memory codec. The getter and the constructor param are wrong as well, probably a matcher issue

Reply:
Fixed in 19ea7d3d586be3cecd60f0f8dd730aca5d634efc.